backend/src/Response/Admin/StoreOwnerSubscription/AdminStoreSubscriptionResponse.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Response\Admin\StoreOwnerSubscription;
+
+use OpenApi\Annotations as OA;
+use Nelmio\ApiDocBundle\Annotation\Model;
+use DateTime;
+
+class AdminStoreSubscriptionResponse
+{
+    public int $id;
+
+    public string $packageName;
+
+    public DateTime $startDate;
+
+    public DateTime $endDate;
+
+    public string|null $status;
+
+    public string|null $note;
+
+    public int $flag;
+    
+    /**
+     * @OA\Property(type="array", property="paymentsFromStore",
+     *     @OA\Items(type="object",
+     *              ref=@Model(type="App\Response\Admin\StoreOwnerPayment\AdminStoreOwnerPaymentResponse"),
+     * ))
+     */
+    public array $paymentsFromStore;
+
+    /**
+     * @OA\Property(type="object", property="total",
+     *              @OA\Property(type="number", property="total"),
+     *              @OA\Property(type="number", property="packageCost"),
+     *              @OA\Property(type="number", property="sumPayments"),
+     *              @OA\Property(type="boolean", property="advancePayment"),
+     * 
+     * )
+     */
+    public array $total;
```

Comment:
I'm afraid that 'array' type may be conflicted with 'object'. As long as you define the type of the variable using OpenAPI annotation, then you do not have to define the type again explicitly (do not have to add 'array" before the var).